sys/net/network_layer/ng_sixlowpan/ctx/ng_sixlowpan_ctx.c

```diff
@@ -79,7 +79,7 @@ ng_sixlowpan_ctx_t *ng_sixlowpan_ctx_lookup_id(uint8_t id)
 
     mutex_lock(&_ctx_mutex);
 
-    if (_still_valid((unsigned int)id)) {
+    if (_valid((unsigned int)id)) {
```

Comment:
Does this really matter here? It's an inline function (:wink:) and I would prefer more readable code over one or two bytes on the stack in case the compiler does not inline the function.